src/com/homework/ChatMember.java

```diff
@@ -1,17 +1,20 @@
 package com.homework;
 
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Instead of using wildcard * consider importing only what your code needs . See more at https://www.codebyamir.com/blog/pitfalls-java-import-wildcards